blockchain/tx_pool.go

```diff
@@ -419,9 +421,10 @@ func (pool *TxPool) reset(oldHead, newHead *types.Header) {
 	pool.currentState = stateDB
 	pool.pendingNonce = make(map[common.Address]uint64)
 	pool.currentBlockNumber = newHead.Number.Uint64()
-	// TODO :  Need to KIP-71 hardfork
-	// TODO : It need to update gas price of tx pool after implemnted baseFee calculation logic.
-	// pool.gasPrice = misc.calculateBaseFee()
+	// It need to update gas price of tx pool after kip71 hardfork
+	if pool.kip71 {
```

Comment:
Is this okay to put here? How about moving this logic after [L465](https://github.com/klaytn/klaytn/pull/1442/files#diff-b4c0a52e4fd03bcfee19698d39bf4eb83280db74b214427a02752f49cac20757R317)?

Reply:
It would have been shifted.. Thanks!!